sdk/keyvault/Azure.Security.KeyVault.Secrets/src/SecretClientBuilderExtensions.cs

```diff
@@ -24,6 +25,8 @@ public static IAzureClientBuilder<SecretClient, SecretClientOptions> AddSecretCl
         /// <summary>
         /// Registers a <see cref="SecretClient"/> instance with connection options loaded from the provided <paramref name="configuration"/> instance.
         /// </summary>
+        [RequiresUnreferencedCode("Binding strongly typed objects to configuration values is not supported with trimming. Use the Configuration Binder Source Generator (EnableConfigurationBindingGenerator=true) instead.")]
```

Comment:
I'm assuming that adding these attributes suppresses the associated warnings since you are explicitly marking the method. Is that correct?